### PR TITLE
Fix gps

### PR DESCRIPTION
**Pull request by danduk82**
This PR adds fix the GPS code to use a (still to be implemented in KADAS) normal `QgsApplication.gpsConnectionRegistry()` to get the GPS signal. Therefore relying on KADAS core to connect to the GPS.

### CODE REVIEW

File: kadasrouting/gui/navigationpanel.py [hunk 1]
```diff
@@ -209,8 +209,8 @@ def hide(self):
         self.stopNavigation()
 
     def updateNavigationInfo(self, gpsinfo):
-        # prevent infinite loop for mocked gps
-        self.gpsConnection.statusChanged.disconnect(self.updateNavigationInfo)
+        # FIXME: prevent infinite loop for mocked gps
+        # self.gpsConnection.statusChanged.disconnect(self.updateNavigationInfo)
```
Comment (danduk82) [inline kadasrouting/gui/navigationpanel.py:213]: @volaya  I am not sure here. Was it correct to comment out this line?

Comment (ismailsunni) [reply kadasrouting/gui/navigationpanel.py:213]: Yes, it's correct. This line was needed for the "GPS" movement by panning the map.

Comment (ismailsunni) [reply kadasrouting/gui/navigationpanel.py:213]: It will need to remove line 230 also since it's no longer needed.

**Review by ismailsunni — APPROVED** [review]
It looks good to me. Never tried a GPS device before, so I assume this should be correct.

But, since it will take time to implement the needed part on Kadas core, I believe we will merge this after Kadas's code has been updated.

**Review by danduk82 — COMMENTED** [review]
thanks @ismailsunni

**Review by ismailsunni — APPROVED** [review]
LGTM

**Pull request merged**